activerecord/lib/active_record/associations.rb

```diff
@@ -1513,6 +1536,22 @@ module ClassMethods
         #   persisted new records placed at the end.
         #   When set to +:nested_attributes_order+, the index is based on the record order received by
         #   nested attributes setter, when accepts_nested_attributes_for is used.
+        # [:before_add]
+        #   Defines a callback that gets triggered before an object is added to the association collection.
+        #
+        #   Read more about {Association callbacks}[#module-ActiveRecord::Associations::ClassMethods-label-Association+callbacks].
```

Comment:
Is there a better way to link to the `Association callbacks` heading in the generated page?

Reply:
`https://edgeapi.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#module-ActiveRecord::Associations::ClassMethods-label-Association+callbacks`

![Screenshot from 2024-06-08 20-22-58](https://github.com/rails/rails/assets/277819/d2d87342-ecc4-4fee-8a20-bd414e8b320b)